locales/pt/pt.json

```diff
@@ -639,16 +639,16 @@
     "Tuvalu": "Tuvalu",
     "Two Factor Authentication": "Autenticação de dois fatores",
     "Two factor authentication is now enabled. Scan the following QR code using your phone's authenticator application.": "A autenticação de dois fatores está ativa. Digitalize o seguinte QR code utilizando a aplicação de autenticação do seu telefone.",
-    "Uganda": "Pai",
-    "Ukraine": "Ucrania",
+    "Uganda": "Uganda",
```

Comment:
in exclude file

Reply:
@st693ava in your case, add words to exclusions here "excludes/pt.php"